code/game/objects/items/cards_ids.dm

```diff
@@ -174,10 +174,9 @@
 	if(!isliving(user))
 		return
 	if(!user.canUseTopic(src, BE_CLOSE, FALSE, NO_TK))
-		return
-	if(user.get_active_held_item() != src)
 		to_chat(user, "<span class='warning'>You must hold the ID in your hand to do this.</span>")

```

Comment:
You must hold the ID in your hand to do this.

Reply:
yeah i was just about to fix this re